SCClassLibrary/Common/Math/SimpleNumber.sc

```diff
@@ -695,20 +695,24 @@ SimpleNumber : Number {
 	// see String:asSecs for complement
 
 	asTimeString { |precision = 0.001, maxDays = 365, dropDaysIfPossible = true|
-		var number, decimal, days, hours, minutes, seconds, mseconds;
+		var number, decimal, days, hours, minutes, seconds, mseconds, isNegative = false;
+		if(this < 0, {
+			isNegative = true;
+		});
 
 		// min value of precision is 0.001; this ensures that we stick to 3 decimal places in the
 		// formatted string.
 		precision = max(precision, 0.001);
 
-		number = this.round(precision);
+		number = this.abs.round(precision);
 		decimal = number.asInteger;
 		days = decimal.div(86400).min(maxDays);
 		days = if(dropDaysIfPossible and: { days == 0 }) {
 			days = ""
 		} {
 			days.asString.padLeft(3, "0").add($:);
 		};
+		if(isNegative, {days = "-" ++ days});
```

Comment:
in general we use trailing closure for `if` in the class library. i've added a style point for that here: https://github.com/supercollider/supercollider/wiki/Code-style-guidelines#trailing-closure-syntax

can you change this one here please?